account_parent/security/account_parent_security.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="group_coas_user" model="res.groups">
+        <field name="name">Show Chart of Account Structure</field>
+        <field name="implied_ids" eval="[(4, ref('account.group_account_manager'))]"/>
+        <field name="users" eval="[(4, ref('base.user_root'))]"/>
+    </record>
+</odoo>
```

Comment:
You should change the logo of the module.

Maybe this one: https://upload.wikimedia.org/wikipedia/commons/9/9c/Categorisation-hierarchy-top2down.svg